numpy/core/src/common/npy_cpu_features.c.src

```diff
@@ -509,10 +510,23 @@ npy__cpu_init_features_arm8(void)
 #include "npy_cpuinfo_parser.h"
 
 __attribute__((weak)) unsigned long getauxval(unsigned long); // linker should handle it
+#elif defined(__FreeBSD__)
+#include <sys/auxv.h>
+#include <stdio.h>
+#include "npy_cpuinfo_parser.h"
+static unsigned long getauxval(unsigned long k)
+{
+    unsigned long val = 0ul;
+    if (elf_aux_info((int)k, (void *)&val, (int)sizeof(val)) != 0)
```

Comment:
```suggestion
    if (elf_aux_info != 0 && elf_aux_info((int)k, (void *)&val, (int)sizeof(val)) != 0) {
```
since elf_aux_info() function appeared in FreeBSD 12.0.

Reply:
more likely `elf_aux_info == 0 || elf_aux_info ...` since this is the error code path. but ok with the rest.